pkg/apisix/schema.go

```diff
@@ -63,7 +63,7 @@ func (sc schemaClient) getSchema(ctx context.Context, name string) (*v1.Schema,
 		)
 	}
 
-	url := sc.url + "/" + name
+	url := sc.url + name
```

Comment:
Why change this? Maybe path.Join is more appropriate.

Reply:
Because I checked logs of E2E test and found there is a duplicated `/`:
![image](https://user-images.githubusercontent.com/26627380/133081137-f7396837-81cc-4c9b-bcb1-0d8499ec04df.png)

Reply:
OK, use `path.Join` is better.

Reply:
It's not suitable to use `path.Join`:
![image](https://user-images.githubusercontent.com/26627380/133257579-6b8e88f1-3f54-47eb-be03-8b4cb5668a2a.png)